conans/test/integration/graph/core/test_version_ranges.py

```diff
@@ -418,6 +418,9 @@ def test_different_user_channel_resolved_correctly():
 
 def test_unknown_options():
     c = TestClient()
-    c.save({"conanfile.py": GenConanfile("pkg", "1.0").with_requirement("lib/[>1.2, <1.4]")})
-    c.run("install .", assert_error=True)
-    assert 'Unrecognized version range option " <1.4" in ">1.2, <1.4"' in c.out
+
+    c.run("graph info --requires=lib/[>1.2,<1.4]", assert_error=True)
```

Comment:
This ``assert_error=True`` is failing for another reason, not the bad syntax.

Reply:
Yes! The error was coming from not being able to resolve the version, but the following assert was still valid :)

I've aded some code to the test to make it resolve just in case